awesomeConfig.cmake

```diff
@@ -398,13 +401,30 @@ add_custom_command(
 
 add_custom_command(
         OUTPUT ${BUILD_DIR}/awesomerc.lua ${BUILD_DIR}/docs/05-awesomerc.md
+        ${BUILD_DIR}/script_files/rc.lua
```

Comment:
This belongs to `OUTPUT`, right?
Should be indented then maybe..

Also for the `COMMAND` lines below then.